### PR TITLE
fix issue with configparser and backports namespace blocking inclusion of nbconvert with IPython

### PR DESCRIPTION
**Pull request by boegel**
fix for #3825

requires ~~https://github.com/hpcugent/easybuild-framework/pull/2194~~, since `nbconvert` 5.1.1 requires that `use_pip` is enabled

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/easyconfigs/c/configparser/configparser-3.5.0-foss-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -12,26 +12,18 @@ toolchain = {'name': 'foss', 'version': '2016a'}
...
-dependencies = [
-    ('Python', '2.7.11'),
-]
-builddependencies = [
-    ('pip', '8.1.2', versionsuffix),
-]
+patches = ['configparser-%(version)s_no-backports-namespace.patch']
+
+dependencies = [('Python', '2.7.11')]
+builddependencies = [('pip', '8.1.2', versionsuffix)]
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/configparser/configparser-3.5.0-foss-2016a-Python-2.7.11.eb]: haven't we got one inside Python?

Comment (boegel) [inline easybuild/easyconfigs/c/configparser/configparser-3.5.0-foss-2016a-Python-2.7.11.eb]: the one included in Python 2.7.11 is too old, pip 8.x is required for `pip install --prefix`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2151.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fd4a38fdea1b5048c4feb9fd18e9c352 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 8 (8 easyconfigs in this PR)
node2523.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bd8a363f61b728bc3a64d0838a39ad14 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2523.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/537695dcb083f059089faebaf13546bc for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**